build.sbt

```diff
@@ -101,6 +103,9 @@ val alleycatsRetry = crossProject(JVMPlatform, JSPlatform)
       "org.scalatestplus" %%% "scalatestplus-scalacheck" % scalaTestPlusVersion % Test,
       "org.typelevel"     %%% "discipline-scalatest"     % disciplineVersion    % Test,
       "org.scalacheck"    %%% "scalacheck"               % scalacheckVersion    % Test
+    ),
+    mimaPreviousArtifacts := Set(
+      "com.github.cb372" %%% "alleycats-retry" % "1.0.0"
```

Comment:
Shouldn't this be `1.1.0` as well?

Reply:
I'm not sure. Don't we want to ensure that every 1.x release is compatible with 1.0.0? Isn't that necessary and sufficient to ensure bincompat?

For the mtl module, 1.0.0 doesn't exist so I made it 1.1.0.

Reply:
I'd assume we'd want every new release to be compatible with the last one. If `1.1.0` is bin-compatible with `1.0.0`, then any version compatible with `1.1.0` is automatically compatible with `1.0.0`, no? In other words compatibility is transitive. The reason to choose the latest version is so that e.g. if a hypothetical `1.2.0` changes something that was added in `1.1.0`, it might very well be compatible with `1.0.0`, but incompatible with `1.1.0`. That make sense?

Reply:
I see. So we'll need to keep this updated every time we release. Would be nice to automate that.